libraries/AP_Scheduler/PerfInfo.h

```diff
@@ -26,6 +36,25 @@ class PerfInfo {
 
     void update_logging();
 
+    // allocate the task info array
```

Comment:
This comment really doesn't add much.  Explaining what we store in it and why would be better.

Reply:
Fixed